examples/tests/rootfs/usr/local/etc/atmos/atmos.yaml

```diff
@@ -19,6 +19,16 @@ base_path: ""
 
 components:
   terraform:
+    # Optional `command` specifies the executable to be called by `atmos` when running Terraform commands
+    # If not defined, `terraform` is used
+    # Examples:
+    # command: terraform
+    # command: usr/local/bin/terraform
+    # command: usr/local/bin/terraform-1.8
+    # command: tofu
+    # command: usr/local/bin/tofu-1.7.1
```

Comment:
It does not make sense to me why we would not have the leading `/` in the examples

Reply:
for `/usr/local/bin`

Reply:
I suspect this is because in your _tests_ you are not running in a container, but the comments send the _wrong_ message to 99% of people who would use this.